packages/terra-button-group/src/ButtonGroup.module.scss

```diff
@@ -30,17 +30,18 @@
       // A z-index needs to be added on focus so that the focus ring does not get hidden behind adjacent buttons
       z-index: 1000;
     }
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none), (hover: hover) and (pointer: fine) {
```

Comment:
what happens if we don't include the ms-high-contrast queries?

Reply:
The interaction media features hover and pointer are not supported on IE and it has been addressed by making use of -ms-high-contrast.